lib/openshift/console_cli_download_console_openshift_io.rb

```diff
@@ -0,0 +1,8 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # @note represents an OpenShift cluster resource about console
+  class ConsoleCliDownloadConsoleOpenshiftIo < ClusterResource
+    RESOURCE = 'console.cli.download.console.openshift.io'
```

Comment:
Are you sure this is `console` or is it `consoles`?

Reply:
It's `console`, not `consoles`
I'd even suggest to use `consoleclidownload.console.openshift.io` and `ConsoleclidownloadConsoleOpenshiftIo` because `consoleclidowload` is a spelling of the resource. It's clear and convenient for others to get what does the whole word `consoleclidowload` mean. 
Now I'm compromised :)

Reply:
Wait, please do not be compromized! :)

`RESOURCE` should reflect the actual object type and API that is used in OpenShift. This must be `consoleclidownloads.console.openshift.io`:
```
🐚 oc --config=/tmp/kubeconfig get consoleclidownloads.console.openshift.io 
No resources found.
```

The name of the class and corresponding short name to use in framework we have a convention for them to be more readable.

So one thing is `RESOURCE` that we need to check OpenShift API about and we cannot change. The other thing is how we type the names in the framework.

Does it now make more sense?